workflow/usage.rst

```diff
@@ -0,0 +1,104 @@
+.. index::
+    single: Workflow; Usage
+
+How to Use the Workflow
+=======================
+
+The workflow component gives you an object oriented way to work with state
```

Comment:
Agreed. I'm moving all discussions on state machines to a separate topic. Thank you.